src/scripts/inverted-index.js

```diff
@@ -0,0 +1,86 @@
+/*
+* InvertedIndex Class
+*/
+class invertedIndex {
+
+  constructor() {
+    this.indexMap = {};
+  }
+
+  sanitize(indexes) {
+    let lowerindex = [];
+    indexes.map((word) => {
+      lowerindex.push(word.toLowerCase().replace(/[!''@#$%^&*,'.]/g, ""));
+    });
+    return lowerindex;
+  }
+
+	/*
+	* Create File Index
+	*/
+  createIndex(fileName, fileContent) {
+    let completeIndex = [];
```

Comment:
**Code Quality - Nitpick**
Inconsistent use of `let` and `const`.
`const` is appropriate in this context as the variable `completeIndex` is not reassigned.
Take a look at our AirBnB [styleguide](https://github.com/airbnb/javascript#references)